6-appsource/cymbal-bank/artifact-registry.tf

```diff
@@ -0,0 +1,55 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# create artifact registry for container images
+resource "google_artifact_registry_repository" "container_registry" {
+  repository_id = local.application_name
+  location      = var.region
+  format        = "docker"
+  description   = "Bank of Anthos docker repository"
+  project       = var.project_id
+
+  provider = google-beta
```

Comment:
Does this (still?) need to be the beta provider?  Didn't see anything beta specific in https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/artifact_registry_repository

Reply:
removed using beta provider